src/apps/Map/components/SearchBar.js

```diff
@@ -111,6 +123,7 @@ class SearchBar extends PureComponent {
               placeholder="Suche einen Ort"
               onChange={this.onChange}
             />
+            {!this.state.inputValue ? <SearchIcon /> : null}
```

Comment:
Not using not ;-) is easier to read IMO.

```suggestion
            {this.state.inputValue ? null : <SearchIcon />}
```